### DISCUSSION TITLE
Workflow expected behavior is not as described

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16172] [General]
So, we have this triggers like 'more than hours before' or 'less than hours before' etc.
Admin designing processes understand those workflows straight forward: cron running through records checking if conditions has been met, than workflow triggers. **Well, actually not** and I understand it would be difficult with tens of millions of records. To trigger above mentioned (and other similar) certain record needs to be **kicked** (for instance edited and saved without changes, or trigger manually). It makes those workflows naming pattern ambiguous. How about adding to the workflow this certain kick somehow, I mean especially to "Only once, when all conditions are met" ?
I want to add that "Execution schedule (defined time intervals)" will kick it, but how about extra large set of data?
Hoping for input in this matter :-)

Have a glorious weekend!

**Comment by GarethHumbel** [comment]
I have been testing workflows on date-time field and found that hours later trigger is not working. I thing this is only on time field? If so, what workflows going to work on date-time field?

**Comment by Marshal4582** [comment]
I just tried following basic workflow:
1. Execution schedule 5 min
2. created time > More than hours before (in this case I entered 1 and checked that created time is indeed more than 1 hour before)

Workflow Not triggering. All perfectly configured. Version 6.3 
It is bug I guess?

Need to add that Manual workflow is just fine.
